crates/catalog/rest/Cargo.toml

```diff
@@ -40,5 +41,7 @@ urlencoding = { workspace = true }
 uuid = { workspace = true, features = ["v4"] }
 
 [dev-dependencies]
+iceberg_test_utils = { path = "../../test_utils" }
```

Comment:
This will be a blocker for our crate release.

Reply:
Any other suggestions? Another approach I come up with is to specify paths for module, like following:
```
#[path="../../test_utils"]
mod iceberg_test_utils;
```

Reply:
https://github.com/apache/incubator-opendal/blob/fc3e48063520f0a3f1270d75940ee29ab773b9f0/core/src/raw/mod.rs#L68-L69